configure.in

```diff
@@ -136,76 +128,72 @@ if test "$enable_openssl" = "yes"; then
              [AC_MSG_FAILURE([can't find openssl >1.0.1 crypto lib])])
```

Comment:
This is now addressed in 6c949b697a0e71039c5ef66e95cf2efde770e2b6, the check is chnaged to >= 1.0.1